certified-connectors/DocTOPDFConnector/apiDefinition.swagger.json

```diff
@@ -0,0 +1,107 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.6",
+    "title": "Doc To Pdf by Spot solutions",
+    "description": "Convert word document to pdf file",
+    "contact": {
+      "name": "Spot Solutions",
+      "url": "https://spotsolutions.com/",
+      "email": "info@spotsolutions.com"
+    }
+  },
+  "host": "powerstore.spotsolutions.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [],
+  "paths": {
+    "/api/doctopdf": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "response status"
+                },
+                "File Name": {
+                  "type": "string",
+                  "description": "file name"
+                },
+                "File Content": {}
+              }
+            }
+          }
+        },
+        "summary": "Doc to Pdf",
+        "operationId": "doctopdf",
+        "description": "convert library word document file to pdf format file",
+        "parameters": [
+          {
+            "name": "body",
+            "x-ms-summary": "filename and file contect in array format",
+            "description": "filename and file contect in array format",
```

Comment:
All the descriptions must be free from grammatical errors. Please review and update all descriptions and summaries.